src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/claimantlip/casestate/ClaimantLiPCaseStateManagment.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user.claimantlip.casestate;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.civil.enums.CaseState;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+
+@Component
+@AllArgsConstructor
```

Comment:
Please use the same approach as #3376

Reply:
In order to avoid overloading the handler with different tasks like triggering the camuda process, and changing the state, which messes up a lot of code in the handler,i.e is the reason  I created a separate service to handle the state of the case, discussed the same with Neeta, agreed with this approach.

Reply:
This is something Tanya was also encouraging to do before she went off. Do yo remember? Keep it separated.
Additionally as a hint it very well suits upder Open/Close principle.
https://medium.com/swift-india/solid-principles-part-2-open-closed-principle-bf0514b824ca